src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_fr.properties

```
@@ -0,0 +1,12 @@
+AbstractFolder.BeingDeleted={0} est en cours de suppression
+AbstractFolder.FailureToStopBuilds=Impossible de suspendre et de stopper {0,choice,1#{0,number,integer} construction|1<{0,number,\
+  integer} constructions} de {1}
+Folder.DisplayName=Dossier
+Folder.DefaultPronoun=\u00c9l\u00e9ment
+Folder.Description=Cr\u00e9e un conteneur qui stocke des objets imbriqu\u00e9s. Utile pour grouper ensemble des \u00e9l\u00e9ments. \
+   Contrairement \u00e0 une vue qui n''est qu''un filtre, un dossier cr\u00e9e un espace de nommage distinct, de sorte que vous \
+  pouvez avoir plusieurs \u00e9l\u00e9ments du m\u00eame nom tant qu''ils se trouvent dans des dossiers diff\u00e9rents.
+
+RelocateAction.permission.desc=N\u00e9cessaire pour d\u00e9placer un job d''un dossier (ou de la racine Jenkins) \u00e0 un autre.
```

maybe `job` -> `projet`.

```suggestion
RelocateAction.permission.desc=N\u00e9cessaire pour d\u00e9placer un projet d''un dossier (ou de la racine Jenkins) \u00e0 un autre.
```

---

Thanks for the feedback!
I have updated my changes accordingly.